### ISSUE TITLE
Bring the promts out separately to allow for translation into other languages.

### ISSUE THREAD

**Issue by konstantin1722** [logancyang/obsidian-copilot#496]
Now there are already several models that work perfectly with languages of other groups (e.g. Japanese, Chinese, Russian). Ideally I would like to be able to edit the built-in promts through the interface or even have several versions in different languages.

In the meantime, please fix the design of the code and put all the promts in a separate file so that translation can be done more easily. Also you can make some kind of method that will give the necessary prompt by name and language code.

Prompts written in the same language as the inputs - works better.

**Comment by logancyang** [comment]
Those built-in prompts are just for demonstration, you can create your custom prompts which are named by you and saved locally.

**Comment by konstantin1722** [comment]
> Those built-in prompts are just for demonstration, you can create your custom prompts which are named by you and saved locally.

Can this be done through the GUI?

**Comment by logancyang** [comment]
@konstantin1722 yes, just use the add/edit/apply/delete custom prompts commands, ideally with shortcuts you can define in Obsidian.
